packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -172,6 +172,30 @@ export const standardLinterfunctions: FunctionItem[] = [
       return true;
     },
   },
+  {
+    functionName: 'existAnyOfFields',
+    function: (
+      element: Element,
+      keys: string[],
+      allowEmpty: boolean,
+      skipIfType?: string,
```

Comment:
This `skipIfType` is a bit confusing to me. It tries to solve a problem already solved by rules `conditions`. Wouldn't it be better to just having a condition for asserting that schema is `schema` element type and only then executing `existAnyOfFields` in particular linting rules?